engines/dragons/configure.engine

```diff
@@ -1,3 +1,3 @@
 # This file is included from the main "configure" script
-# add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps]
-add_engine dragons "Blazing Dragons" yes "" "" "16bit"
+# add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps] [components]
+add_engine dragons "Blazing Dragons" yes "" "" "16bit" "mt32emu"
```

Comment:
How useful is MT32 support here? The engine expects to be provided with a MIDI driver that supports soundfonts constructed in memory, and I don't know how effectively it can fall back to general MIDI.

Reply:
Yes, fixed.